build-tools/mono-runtimes/mono-runtimes.projitems

```diff
@@ -15,7 +15,8 @@
       <RanLib>$(_ArmRanLib)</RanLib>
       <Strip>$(_ArmStrip)</Strip>
       <ConfigureFlags>--host=armv5-linux-androideabi $(_TargetConfigureFlags)</ConfigureFlags>
-      <OutputRuntime>libmonosgen-2.0.so</OutputRuntime>
+      <OutputRuntime>libmonosgen-2.0</OutputRuntime>
+      <OutputRuntimeExtension>so</OutputRuntimeExtension>
```

Comment:
looking at the well known msbuild properties [1]

%(Filename)  = Filename without the extension
%(Extension) = extension :)

so following that 

%(OutputRuntimeFilename)
%(OutputRuntimeExtension)

seems to be reasonable.

[1] https://msdn.microsoft.com/en-us/library/ms164313.aspx